keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepository.kt

```diff
@@ -172,6 +174,10 @@ interface ResourceRepository : PeriodicallyCheckedRepository<Resource<out Resour
     return first() is ResourceActuationPaused
   }
 
+  private fun List<ResourceEvent>.isVetoed(): Boolean {
```

Comment:
FYI @erikmunson this will never be hit right now (so no resource will have a status of `vetoed`). This is for when we add other vetos.